lib/gui/modules/analytics.js

```diff
@@ -39,6 +40,22 @@ const analytics = angular.module(MODULE_NAME, [
   require('../models/settings')
 ]);
 
+/**
+ * @summary Get OS architecture
+ *
+ * @description
+ * We need this because node's os.arch() returns the process architecture
+ * See: https://github.com/nodejs/node-v0.x-archive/issues/2862
+ *
+ * @return {String} OS architecture
+ */
+const getOSArch = () => {
+  if (os.arch() !== 'ia32') {
+    return os.arch();
+  }
+  return arch() === 'x86' ? 'ia32' : 'x64';
```

Comment:
I don't think we should use this module, given it doesn't seem to handle arm correctly (based on the docs), which is a target we will be releasing for GNU/Linux soon.